pic/app.js

```diff
@@ -0,0 +1,21 @@
+const newImg = document.querySelector('div.image');
+const img = document.createElement('img');
+newImg.appendChild(img);
+let url = "https://source.unsplash.com/1600x900";
+const boot = document.querySelector('button')
+const wi = document.querySelector('.width');
+const hei = document.querySelector('.height');
+
+const onclick = function (){
+    const w = wi.value;
+    const h = hei.value;
+    url+="?"+Math.floor(Math.random()*1000);
```

Comment:
I am not sure why you used a Math.random in the url?
```suggestion
  url = "https://source.unsplash.com/" + w + "x" +  h ;
```
This will return an Image with the same width and height that you provide it in the URL.